openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/TierCard.css

```diff
@@ -0,0 +1,6 @@
+.ant-popover-arrow {
+  display: none !important;
+}
+.ant-popover {
+  width: 760px;
+}
```

Comment:
We can't override it globally. Instead, we can try `Dropdown` component to remove the arrow.
For width you can try this prop: `overlayStyle` refer here: https://ant.design/components/dropdown/#Dropdown